util/sync_point.h

```diff
@@ -60,6 +60,12 @@ namespace rocksdb {
 
 class SyncPoint {
  public:
+  enum class Mode {
```

Comment:
Don't we need another one for sync-points-only mode?

Reply:
Hm, there's currently a global int variable (`rocksdb_kill_odds`) that disables killpoints when set to zero. That's why our unit tests don't get killed when `Mode::kAll` is specified.

Thinking about it more, maybe we should just separate SyncPoint and KillPoint into different classes. They don't really seem to have much overlap... Let me try that way.